production/helm/loki-stack/values.yaml

```diff
@@ -9,6 +9,10 @@ fluent-bit:
 
 grafana:
   enabled: false
+  service:
+    type: LoadBalancer
+    port: 3000
```

Comment:
This seems weird to use a loadbalancer and then expose it at `3000`. If we're using an external loadbalancer, shouldn't we use `80:3000`?